test_requirements.txt

```diff
@@ -1,22 +1,26 @@
-pytest-pep8
-codecov
-
-h5py
-requests
+pytest-pep8==1.0.6
+codecov==2.0.22
+h5py==2.10.0
+requests==2.23.0
+statsmodels==0.11.0
+cma==2.7.0
 
-keras
-mxnet
-#torch>=1.2.0
-tensorflow
+#ART supported Frameworks
+tensorflow==2.1.0
+keras==2.3.1
+# tensorflow==1.15.2
+# keras==2.2.5
```

Comment:
This breaks the testing on Travis where we are running with different version combination of TensorFlow and Keras. Currently all test runs are running with the same combination of `tensorflow==2.1.0` and `keras==2.3.1`. Is this change required for the Docker image?

Reply:
no that's fine, it shouldn't impact the docker image. I can keep tensorflow and keras without any version number if that helps fix travis, and instead just specify them within the dockerfile directly